holmes/plugins/toolsets/teams/teams.py

```
@@ -0,0 +1,446 @@
+                TeamsSearchUsers(self),
+                TeamsCreateChat(self),
+                TeamsSendChatMessage(self),
+                TeamsListChats(self),
+                TeamsGetChatMessages(self),
+            ],
+        )
+
+    def prerequisites_callable(self, config: dict[str, Any]) -> Tuple[bool, str]:
+        try:
+            self.config = TeamsConfig(**config)
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)